# A representation of pose in free space, composed of position (x, y, z) and orientation (only yaw) in degrees
Point pos
Rotation rot